test/test_jit.py

```diff
@@ -295,6 +295,32 @@ def getExportImportCopy(self, m, also_test_file=True, map_location=None):
 
         return result
 
+    def getExportImportCopyWithPacking(self, m, also_test_file=True, map_location=None):
```

Comment:
What's the "packing" part? How does it interact with regular `torch.jit.save`? Do those methods have to be called manually like in here?